### ISSUE TITLE
Anime Matrix Text Mode

### ISSUE THREAD

**Issue by Subtilizer28** [seerge/g-helper#706]
**Is your feature request related to a problem? Please describe.**
its not a problem but would be great to have text displayed on the anime matrix like the armory crate

**Describe the solution you'd like**
an option of text and a box to enter the text to be displayed

**Describe alternatives you've considered**
a picture with the text can be made but would be tough to adjust to the anime matrix and wouldnt work on larger texts

**Additional context**

**Comment by seerge** [comment]
@Subtilizer28 hello, I was planning to extend anime-matrix features with a small editor somewhere soon. It's one of the features for voting https://github.com/seerge/g-helper/discussions/659

**Comment by GibStorm** [comment]
@seerge could you then also add a mappable hotkey for using  to toggle the activation of the Anime Matrix? It sometimes gets stuck in "on" mode on my G14 laptop while going into standby if I close the lid for example. Then I have to go to G-Helper and toggle the "turn off on battery" manually a couple of times to ensure that it's off. Having the hotkey would enable me to quickly toggle it off (especially of the OSD would show me the active state while toggling).

**Comment by flankedgonerogue** [comment]
Yeah the airplane mode key is pretty useless, using it as a AniMe matrix toggle would be nice.

**Comment by KelseyHigham** [comment]
in the meantime, it's possible to make a text GIF with this template: https://github.com/seerge/g-helper/issues/1648#issuecomment-1826224297

**Comment by TheTommyUknow** [comment]
> @Subtilizer28 hello, I was planning to extend anime-matrix features with a small editor somewhere soon. It's one of the features for voting #659

It's been a year, when will you do it g

**Comment by TechNo1geek** [comment]
Yep, Still waiting on it Aswell ^^ looking forward to it

**Comment by Scarlaid** [comment]
bumping this because this is (might be) the last one feature for the Matrix to be added 👍

**Comment by seerge** [comment]
@ashtonmths @GibStorm @flankedgonerogue @wasokeli @TheTommyUknow @TechNo1geek @Scarlaid @Mbanka4 @alphafortytwo @thduc87 @Secreto31126 @vTusharr @joex92 @Robin-7921 @pat-CIMAR-UF

Hello everyone,

I have finally found time and possibility to extend Matrix options, and added separate mode for Text rendering. It allows to enter up to 2 independent text blocks in own fonts / sizes + make text moving if needed. 

Try this build and let me know if it works for you

[GHelper.zip](https://github.com/user-attachments/files/30121672/GHelper.zip)

<img width="1536" height="1244" alt="Image" src="https://github.com/user-attachments/assets/efabc12e-ae99-447c-89dc-67cb4b405d3d" />

Same build also includes new "preview" UI for Picture / Clock / Audio modes :)

Thanks!

**Comment by vTusharr** [comment]
Yes indeed it works well ! Thanks fo the build!

**Comment by TechNo1geek** [comment]
mine died (actual iGPU) :( moved to mac sadly

**Comment by ashtonmths** [comment]
The adjusting of image works perfectly and it was needed
text mode works perfectly as well

thank you for working on this @seerge!!

**Comment by seerge** [comment]
Live in latest release https://github.com/seerge/g-helper/releases
